cfpb/cfgov-refresh
# Makes more than one skip-link per page possible

Find a housing counselor page has two skip links. The default one in the header and one to skip over the map to the list of results. This PR makes the skip link CSS tied to a class instead of an ID so more than one skip link can be added to a page.

## Changes

- Converts `skip-link` ID to class.
- Adds skip link to find a housing counselor page for skipping over the map.
- Adds 10px margin to skip link.

## Testing

1. Make sure `HUD_TOOL_IMPROVEMENTS` is set to boolean true.
1. Visit http://localhost:8000/find-a-housing-counselor/ and tab through the page to show the "Skip to results" skip link. Shift+tab to move back to the "Skip to main content" skip link.
1. http://localhost:8000/paying-for-college/manage-your-college-money/ and tab through the page to show the "Skip to main content" skip link.

## Screenshots

<img width="326" alt="screen shot 2018-11-21 at 3 02 12 pm" src="https://user-images.githubusercontent.com/704760/48865726-764a5a00-ed9e-11e8-91b7-8544b0b13032.png">

<img width="509" alt="screen shot 2018-11-21 at 3 02 19 pm" src="https://user-images.githubusercontent.com/704760/48865727-764a5a00-ed9e-11e8-8d84-71e7f060c11e.png">
